src/snowflake/cli/api/exceptions.py

```diff
@@ -229,3 +229,8 @@ def __init__(self, show_obj_query: str):
         super().__init__(
             f"Received multiple rows from result of SQL statement: {show_obj_query}. Usage of 'show_specific_object' may not be properly scoped."
         )
+
+
+class FeatureNotEnabledError(RuntimeError):
+    def __init__(self, feature_name: str):
+        super().__init__(f"The feature '{feature_name}' is not enabled.")
```

Comment:
Streaming logs from spcs containers is disabled. Set the following field in your configuration to enable it 'ENABLE_SPCS_LOG_STREAMING'.